hardware/pulser_dummy.py

```diff
@@ -221,7 +221,7 @@ def get_constraints(self):
         activation_config['config9'] = frozenset({'a_ch2', 'a_ch3'})
         constraints.activation_config = activation_config
 
-        constraints.sequence_option = SequenceOption.OPTIONAL
+        constraints.sequence_option = SequenceOption.FORCED
```

Comment:
Maybe not a good idea. How about making it a `ConfigOption` for dummy.

Reply:
good point, I like it.

Please check again.